### PR TITLE
DB query & import logic tweak

### PR DESCRIPTION
**Pull request by ashchan**
Mainly for query/insert under `Indexer` module.

### CODE REVIEW

File: packages/neuron-wallet/src/database/chain/ormconfig.ts [hunk 1]
```diff
@@ -26,7 +29,7 @@ const connectOptions = async (genesisBlockHash: string): Promise<SqliteConnectio
   const database = env.isTestMode ? ':memory:' : dbPath(genesisBlockHash)
 
   const logging: boolean | ('query' | 'schema' | 'error' | 'warn' | 'info' | 'log' | 'migration')[] =
-    process.env.SHOW_CHAIN_DB_LOG && (env.isDevMode || env.isTestMode) ? true : ['warn', 'error']
+    (env.isDevMode || env.isTestMode) ? ['warn', 'error', 'log', 'info', 'schema', 'migration'] : ['warn', 'error']
```
Comment (classicalliu) [inline packages/neuron-wallet/src/database/chain/ormconfig.ts]: Default to show logs except `query` in dev and test mode?

Comment (ashchan) [reply packages/neuron-wallet/src/database/chain/ormconfig.ts]: Yes.

Comment (ashchan) [reply packages/neuron-wallet/src/database/chain/ormconfig.ts]: Combined with `maxQueryExecutionTime: 30` in `ormconfig.ts`.

Comment (classicalliu) [reply packages/neuron-wallet/src/database/chain/ormconfig.ts]: OK, I means will see long logs when running in dev / test mode.

**Pull request merged**